### PR TITLE
Naming cleanup

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

**Comment by AronNovak** [comment]
@adamhstewart Can we double-check why Travis doesn't kick in?

**Comment by AronNovak** [comment]
@anvmn I'd do not merge it yet, before seeing Travis on that.

**Comment by AronNovak** [comment]
@anvmn CircleCI is happily building this, let's see.

**Comment by AronNovak** [comment]
@anvmn Do we wait for a specific release to unblock this?

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.module
Comment (anvmn) [inline server/hedley/modules/custom/hedley_migrate/hedley_migrate.module]: @AronNovak why is this removed?

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.module
Comment (anvmn) [inline server/hedley/modules/custom/hedley_migrate/hedley_migrate.module]: @AronNovak why is this removed?

**Review by anvmn — COMMENTED** [review]
Not sure why migration code is removed

**Comment by anvmn** [comment]
@AronNovak ^^

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.module
Comment (AronNovak) [reply server/hedley/modules/custom/hedley_migrate/hedley_migrate.module]: As it's a non-used function and contains the string that we purge, it seemed the best choice to get rid of it.

File: server/hedley/modules/custom/hedley_migrate/hedley_migrate.module
Comment (AronNovak) [reply server/hedley/modules/custom/hedley_migrate/hedley_migrate.module]: Same thinking here. If it's still in use, let me know where and how it's invoked.

**Comment by anvmn** [comment]
@AronNovak 

These functions are used to extend GeoLocation areas project supports.
Example to this is https://github.com/Gizra/ihangane/pull/2770.
The process is semi automatic, meaning that backend code generates Elm code that is transfered to ELM file.
The code must be preserved. I think that replacing `Ihangane` with `Eheza` is the best route of action.

**Review by anvmn — APPROVED** [review]
:+1: Will MWG

**Pull request merged**
